pkgs/development/python-modules/pdfquery/default.nix

```
@@ -0,0 +1,46 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  cssselect,
+  chardet,
+  lxml,
+  pdfminer-six,
+  pyquery,
+  roman,
+  six,
+}:
+
+buildPythonPackage rec {
+  pname = "pdfquery";
+  version = "0.4.3";
```

```suggestion
  version = "0.4.3";
  pyproject = true;
```
Any reason for not adding this?

---

Doing so breaks the build for me. I know the manual says that pyproject will fall back to setuptools, but it doesn't seem to work and I was unable to track down why.

---

What is the error? Isn't it just a matter of adding `setuptools` to `build-system`?